src/graphql/review/submitTechnicalReview.graphql

```diff
@@ -1,30 +1,12 @@
-mutation submitTechnicalReview(
-  $proposalPk: Int!
-  $timeAllocation: Int
-  $comment: String
-  $publicComment: String
-  $status: TechnicalReviewStatus
-  $submitted: Boolean!
-  $reviewerId: Int!
-  $files: String
+mutation submitTechnicalReviews(
```

Comment:
Does it matter that this mutation's name doesn't match the filename anymore?

Reply:
I will fix this.